autonormalize/autonormalize.py

```diff
@@ -143,7 +150,7 @@ def auto_normalize(df):
 
 def normalize_entity(es, accuracy=0.98):
```

Comment:
Not sure if we want to update this from `normalize_entity` to `normalize_dataframe` as well?

Reply:
I actually think it should be normalize entityset, but I didn't want to change the public API